src/header.ts

```diff
@@ -0,0 +1,122 @@
+import {Flag, flagKeys, keys} from './util';
+import {FontWeight, TitleOrient} from './vega.schema';
+
+export const HEADER_TITLE_PROPERTIES_MAP = {
+  titleAlign: 'align',
+  titleAngle: 'angle',
+  titleBaseline: 'baseline',
+  titleColor: 'fill',
+  titleFont: 'font',
+  titleFontSize: 'fontSize',
+  titleFontWeight: 'fontWeight',
+  titleLimit: 'limit',
+  titleX: 'x',
+  titleY: 'y',
+};
+
+export const HEADER_LABEL_PROPERTIES_MAP = {
+  labelAngle: 'angle',
+  labelColor: 'fill',
+  labelFont: 'font',
+  labelFontSize: 'fontSize',
+  labelLimit: 'limit',
+  labelOverlap: 'overlap',
```

Comment:
Can you really support labelOverlap here?  I suspect that you can't.

Reply:
I don't think so. This will be a problem when you create a grouped bar chart with row or column.

Reply:
So I should remove this?

Reply:
Yes. Make sure you only include properties that work in Vega.